tests/assets/eks_nodeGroup_LaunchTemplate.yaml

```diff
@@ -0,0 +1,29 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Create an EKS Node Group Launch Template
+Resources:
+  NodeGroupLaunchTemplate:
+    Type: AWS::EC2::LaunchTemplate
+    Properties:
+      LaunchTemplateName: !Ref LaunchTemplateName
+      LaunchTemplateData:
+        InstanceType: t3.medium
```

Comment:
this won't work, because we need mixed instance fleet, otherwise we would run into icing issues.

Reply:
Removed instances from LT